src/SystemPaths.cc

```diff
@@ -165,20 +203,43 @@ std::string SystemPaths::FindSharedLibrary(const std::string &_libName)
 void SystemPaths::SetFilePathEnv(const std::string &_env)
 {
   this->dataPtr->filePathEnv = _env;
+  std::string result;
+
   if (!this->dataPtr->filePathEnv.empty())
   {
     this->ClearFilePaths();
-    std::string result;
     if (env(this->dataPtr->filePathEnv, result))
     {
       this->AddFilePaths(result);
     }
   }
+  // TODO(CH3): Deprecated. Remove on tock.
+  else if (!this->dataPtr->filePathEnvDeprecated.empty())
+  {
+    this->ClearFilePaths();
+    if (env(this->dataPtr->filePathEnvDeprecated, result))
+    {
+      gzwarn << "Finding files using deprecated environment variable "
+             << "[" << this->dataPtr->filePathEnvDeprecated
+             << "]. Please use [" << this->dataPtr->filePathEnv
+             << "] instead." << std::endl;
+      this->AddFilePaths(result);
+    }
+  }
 }
 
 /////////////////////////////////////////////////
 std::string SystemPaths::FilePathEnv() const
 {
+  // TODO(CH3): Deprecated. Remove on tock.
+  if (this->dataPtr->filePathEnv.empty())
+  {
+    gzwarn << "Returning deprecated file path environment variable "
```

Comment:
This warning will come up if you have no env var set at all, which we don't want. 

I'm wondering if we can ever get to this situation though. The only way to change `filePathEnv` is through `SetFilePathEnv`. From the moment the user calls that, I think we could just empty `filePathEnvDeprecated`, because the user is using a custom variable and doesn't care about the defaults, deprecated or not.

Reply:
Unfortunately `SetFilePathEnv()` is called internally, so it's hard to distinguish a user-originating call.

But removing this warning makes sense!

Reply:
https://github.com/gazebosim/gz-common/pull/365/commits/cb5f31e2958f927c0bcab1db07114fd24df82668